src/main.py

```diff
@@ -7,14 +7,14 @@
 
 #creates new table for usernames and passwords
 def create_table():
-    query = """CREATE TABLE IF NOT EXISTS Username(username TEXT, password TEXT,firstname TEXT,lastname TEXT)"""
+    query = """CREATE TABLE IF NOT EXISTS Username(username TEXT, password TEXT,firstname TEXT, lastname TEXT, logedin INTEGER)"""
```

Comment:
We can also set the default value of `logedin` to be false/zero. Up to you though.

Reply:
yeah that works too but false would need to be a string since sqlite does not have a "BOOL" type for columns like MySQL or other database languages